### PR TITLE
handle indeed edge case, link to job-pipeline

### PR DESCRIPTION
**Pull request by jorgesierra**
- Add app version to about page
Screen Shot 2020-09-17 at 10 52 15

### CODE REVIEW

File: dist/manifest.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "Teal's Job Tracker",
-    "version": "0.0.4",
+    "version": "0.0.5",
```
Comment (keithmgould) [inline dist/manifest.json:3]: thanks for this.
non-blocker: 
0. is it possible to add this to to the about page in some discreet way?
1. Is it possible to have single place to change version that would change both manifest and and the place on the about page?

File: src/ui/components/job-saved-success/job-saved-success.tsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ const JobSavedSuccess = (props: IJobSavedSuccess) => {
             data.id && 
             <Col span={18}>
               <div>{data.company} {data.role}</div>
-              <div>Job saved. <a target="_blank" href={`${WEB_CLIENT_URL}job-tracker/${data.id}`}>View now &#8594;</a></div>
+              <div>Job saved. <a target="_blank" href={`${WEB_CLIENT_URL}job-pipeline/${data.id}`}>View now &#8594;</a></div>
```
Comment (keithmgould) [inline src/ui/components/job-saved-success/job-saved-success.tsx:27]: 🥳

**Review by keithmgould — APPROVED** [review]
left a NB question, but if its too time consuming lets def push out as is.

File: dist/manifest.json (hunk 1 above)
Comment (jorgesierra) [reply dist/manifest.json:3]: Looks like it can be done ✌️ 

`var manifestData = chrome.runtime.getManifest();
console.log(manifestData.version);`

Comment (keithmgould) [reply dist/manifest.json:3]: oh sweet. The main use-case is actually going to be to help members, not for debugging. So I like the log, but also hoping we can get it into the about section so we can ask members to tell us the version number when things don't work as expected. I think we don't have to go crazy thinking about styling, if you can think if a small place to put it in about for now.

**Review by ohaicristina — APPROVED** [review]
this works! might make some adjustments with this story: https://www.pivotaltracker.com/story/show/174780753 but lookin' good 😎 

and now  we can close this one! https://www.pivotaltracker.com/story/show/174410847 🥳

**Pull request merged**